client/src/JobPaginator.test.js

```diff
@@ -14,39 +14,42 @@ jest.mock("./JobPaginatorZoomToggle", () => () => "JobPaginatorZoomToggle");
 describe("JobPaginator", () => {
     let props;
     beforeEach(() => {
-        jest.spyOn(JobPaginatorState, "getJob").mockResolvedValue({
-            order: [
-                {
-                    filename: "test1",
-                    label: "test2",
-                },
-            ],
-        });
-
-        jest.spyOn(JobPaginatorState, "getStatus").mockResolvedValue({
-            file_problems: {
-                deleted: [],
-                added: [],
-            },
-        });
+        global.fetch = jest.fn(() =>
+            Promise.resolve({
+                json: () =>
+                    Promise.resolve({
+                        order: "testOrder",
```

Comment:
It might be more realistic, especially in light of #89, for the order fixture to be an array rather than a string.